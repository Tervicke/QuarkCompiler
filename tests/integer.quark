int p = 42
int n = -1
print(p)
print(n)
int expr = 3 * 45 + 15 
print(expr)
expr = 3 * (45 + 15)
print(expr)
