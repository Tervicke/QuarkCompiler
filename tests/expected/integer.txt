42
-1
150
180
